pkg/applicationserver/io/packages/packages.go

```diff
@@ -142,7 +142,7 @@ func (s *server) NewSubscription() *io.Subscription {
 	go func() {
 		for {
 			select {
-			case <-s.ctx.Done():
+			case <-sub.Context().Done():
```

Comment:
The `Up()` channel used before is available as long as the `sub` is alive (`sub.Context().Done()` is not closed)